template/src/core/network/globalNetworkModels.d.ts

```diff
@@ -0,0 +1,24 @@
+interface ListRequestResponse<Result> {
+    count: number;
+    next: null | string;
+    previous: null | string;
+    results: Result[];
+  }
+  
+  type ListRequestParams<Ordering = string> = Partial<{
+    ordering: Ordering;
+    limit: "all" | number;
+    offset: number;
+    search: string;
+  }>;
+  
+  interface HipoApiErrorDetailShape {

```

Comment:
Is it same type with `Record<string, undefined | any>` ?
If it is we may directly use it in HipoApiErrorShape["detail"] for the sake of simplicity.
what do you think?

Reply:
Yeah I think that makes sense 👍